### DISCUSSION TITLE
Can we Set minimum size for cluster ?

### DISCUSSION THREAD

**Discussion by hengseyha** [inocan-group/vue3-google-map#230] [Q&A]
Can you please help me set the minimum cluster size?

**Comment by HusamElbashir** [comment]
Not sure what you mean @hengseyha . Maybe you can elaborate.

**Comment by hengseyha** [comment]
I mean that I want to show the icon cluster when the map icon is close together by a specific amount. example 100 map icons.

**Comment by HusamElbashir** [comment]
We use `@googlemaps/markerclusterer` to create these clusters. I suppose you can create a [custom clustering algorithm](https://googlemaps.github.io/js-markerclusterer/interfaces/MarkerClustererOptions.html#algorithm) and use that but I'm afraid this feature isn't supported out of the box.

**Comment by hengseyha** [comment]
@HusamIbrahim I cannot make it works, can you please help to provide me some clue to do this?
